plone/app/contenttypes/browser/folder.py

```diff
@@ -165,7 +166,7 @@ def tabular_fields(self):
         ret.append('Type')
         if self.show_about:
             ret.append('ModificationDate')
-        return ret
+        return [(column, COLUMNS_TITLE.get(column, column)) for column in ret]
```

Comment:
To not change the return type of the `tabular_fields` property I would introduce a new method:
```
    def get_tabular_field_label(self, field):
        return COLUMNS_TITLE.get(field, field))
```
and replace `python:field[1]` with `python:view.get_tabular_field_label(field)` in the page template.

Reply:
You're right. I used just `tabular_field_label` without `get_` to be similar to `tabular_fielddata`